src/frontend/src/features/ui/components/service/ServiceBlock.tsx

```diff
@@ -223,25 +234,13 @@ export const ServiceBlock = (props: ServiceBlockProps) => {
         )}
       </div>
       <div className="dc__service__block__body">
-        {!!props.service.description && (
+        {!!props.service.description && !props.service.subscription && (
```

Comment:
Why? We got empty cards if we have a sub now: 
<img width="796" height="235" alt="Capture d’écran 2025-12-12 à 13 10 23" src="https://github.com/user-attachments/assets/a7ebf53b-0b09-4b74-bbab-856ddd25a87c" />